app-android-xamarin/Moodis/Moodis/Network/Face/Face.cs

```diff
@@ -135,10 +132,37 @@ public async Task<IList<Person>> IdentifyPersons(string imageFilePath, Action<Li
                     }
                 }
             }
-
             return personsList;
         }
 
+        //returns true if one or more users have accounts false otherwise.
+        public async Task<Boolean> MultipleAccounts(string imageFilePath, Action<List<DetectedFace>> callback)
```

Comment:
Wouldn't it be just enough to check if the indentified people list is empty or not? Like in the image below
![image](https://user-images.githubusercontent.com/29150754/70067319-bde10400-15f6-11ea-951b-40d1f5b80b2c.png)

Reply:
Yeah that would be a lot better. Can't believe i didn't see it earlier.